pkg/client/informers/informers_generated/internalversion/factory.go

```diff
@@ -68,12 +71,22 @@ func (f *sharedInformerFactory) Start(stopCh <-chan struct{}) {
 	f.lock.Lock()
 	defer f.lock.Unlock()
 
+	if f.started {
+		glog.Warning("Invalid attempt to try to start the shared informer factory multiple times. This is a no-op but you should remove this invocation")
```

Comment:
technically, this invocation might be the "good" one, and the first invocation might be the one that needs removing. worth capturing the callstack that actually started it and presenting both here? then you could confidently say one of them should be removed

Reply:
How does this look?

```
W0829 12:18:05.820645    3014 factory.go:76] Invalid attempt to try to start the shared informer factory multiple times. Original Start() call stack:
goroutine 1 [running]:
k8s.io/kubernetes/vendor/k8s.io/client-go/informers.(*sharedInformerFactory).Start(0xc4202597a0, 0xc420259740)
        /home/vagrant/go/src/k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:94 +0x22a
main.main()
        /home/vagrant/go/src/k8s.io/kubernetes/andy/andy.go:19 +0x141

W0829 12:18:05.820758    3014 factory.go:79] This call stack:
goroutine 1 [running]:
k8s.io/kubernetes/vendor/k8s.io/client-go/informers.(*sharedInformerFactory).Start(0xc4202597a0, 0xc420259800)
        /home/vagrant/go/src/k8s.io/kubernetes/vendor/k8s.io/client-go/informers/factory.go:78 +0x428
main.a(0x1c96c20, 0xc4202597a0)
        /home/vagrant/go/src/k8s.io/kubernetes/andy/andy.go:12 +0x57
main.main()
        /home/vagrant/go/src/k8s.io/kubernetes/andy/andy.go:20 +0x15b
```

Reply:
hmm... actually... we start shared informers twice in the controller manager, once so the service account controllers can get informed, and once after all the other controllers register. auto-starting in `Informer()` means the first controller to set up dictates the resync, which is not idea.